app/assets/stylesheets/admin/layouts.scss

```diff
@@ -0,0 +1,32 @@
+// Place all the styles related to the admin/layouts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.layout-area {
+  background-color: #eee;
+  height: 600px;
+}
+
+.desk,
+.grouping {
+  height: 50px;
+  width: 50px;
+  display: inline-block;
+  position: absolute;
+
+  color: white;
+  font-weight: bold;
+  overflow: hidden;
+
+  padding: 4%;
+}
+
+.desk {
```

Comment:
Should we put the styling of these classes in different files? In case we opt for more involved styling, i.e svgs, etc

Reply:
As in a `desk.scss` file? 

I think that's generally a good idea. At this point I think it might be overkill, since there's only 2 fields in the class, especially since doing it later would be just as easy.